consensus/istanbul/announce/util.go

```diff
@@ -0,0 +1,23 @@
+package announce
+
+import (
+	"fmt"
+
+	"github.com/celo-org/celo-blockchain/consensus/istanbul"
+	"github.com/celo-org/celo-blockchain/p2p/enode"
+)
+
+// DecryptAndParseEnodeURL decrypts an encrypted enodeURL with the given ecdsa key pair, and parses the
+// resulting Node.
+func DecryptAndParseEnodeURL(ecdsa *istanbul.EcdsaInfo, encEnodeURL []byte) (*enode.Node, error) {
+	enodeBytes, err := ecdsa.Decrypt(encEnodeURL)
+	if err != nil {
+		return nil, fmt.Errorf("Error decrypting endpoint. err=%v. encEnodeURL: '%v'", err, encEnodeURL)
```

Comment:
we don't really follow this entirely, but we should start somewhere. Error codes are lowercase.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Ah, this is my bad then. We can make a spring cleaning PR fixing all of these